src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -2409,7 +2409,22 @@ otError RadioSpinel<InterfaceType, ProcessContextType>::SetChannelMaxTransmitPow
 template <typename InterfaceType, typename ProcessContextType>
 otError RadioSpinel<InterfaceType, ProcessContextType>::SetRadioRegion(uint16_t aRegionCode)
 {
-    return Set(SPINEL_PROP_PHY_REGION_CODE, SPINEL_DATATYPE_UINT16_S, aRegionCode);
+    otError error;
+
+    error = Set(SPINEL_PROP_PHY_REGION_CODE, SPINEL_DATATYPE_UINT16_S, aRegionCode);
+
+    if (error == OT_ERROR_NONE)
+    {
+        otLogInfoPlat("Set region code \"%c%c\" successfully", static_cast<char>(aRegionCode >> 8),
```

Comment:
```suggestion
        otLogNotePlat("Set region code \"%c%c\" successfully", static_cast<char>(aRegionCode >> 8),
```
May use `NOTE` log level if this log is infrequent ? 

Thoughts? @zhanglongxia @librasungirl @bukepo

Reply:
Updated

Reply:
agree.